app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='my-3'>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+
+    <div class="field border-bottom" style="height: 50px;">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Name', class: 'form-control border-0 rounded-0 w-100 h-100'  %>
+    </div>
+
+    <div class="field border-bottom" style="height: 50px;">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email', class: 'form-control border-0 rounded-0 w-100 h-100'  %>
+    </div>
+
+    <div class="field border-bottom" style="height: 50px;">
+      <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password', class: 'form-control border-0 rounded-0 w-100 h-100'  %>
+    </div>
+
+    <div class="field" style="height: 50px;">
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: 'Confirm Password', class: 'form-control border-0 rounded-0 w-100 h-100'  %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up" %>
+    </div>
+    <%= render "devise/shared/error_messages", resource: resource %>
+</div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [Optiona and Important] on your hosted app, I can not sign up I guess it is a problem with render, kindly check your build, but your app works fine on local and all requirements have been met :+1: 
![Screenshot from 2023-02-25 17-40-52](https://user-images.githubusercontent.com/66097371/221369007-bb2678e6-80a8-4f5f-881e-8b1896d362a4.png)